playbooks/installer/ospd/main.yml

```diff
@@ -58,6 +58,8 @@
   roles:
       - {role: installer/ospd/overcloud/storage, when: "installer.storage is defined"}
       - {role: installer/ospd/overcloud/network, when: "installer.network is defined and installer.network.isolation.enable == 'yes'"}
+      - {role: installer/ospd/overcloud/swift, when: "installer.swift is defined"}
```

Comment:
There is no need for a dedicated role.
please move this under "installer/ospd/overcloud/storage", see how it does the configurations of different storage backends and add swift there